config/htcondor-ce.spec

```diff
@@ -58,6 +57,21 @@ Requires: /usr/bin/unshare
 %description
 %{summary}
 
+%package core
+Group: Applications/System
+Summary: Core configuration for HTCondor-CE
+
+Provides:  condor-ce = %{version}
+Provides:  %{name}-master = %{version}-%{release}
+
+Requires: condor
+Requires: %{name}-core-client = %{version}-%{release}
+
+Obsoletes: condor-ce < 0.5.4
```

Comment:
Again, you should be careful with this so you don't replace condor-ce with htcondor-ce-core for installations that really need the OSG bits.

Reply:
The `Obsoletes`/`Provides` are from 2012 and nothing in our 3.4 repos require `condor-ce-*` so I think we can probably drop them entirely - at the same time I don't think there's any harm in keeping them around